config/config.go

```diff
@@ -88,6 +89,9 @@ func Parse(c CfgMap) error {
 				}
 
 				reflect.ValueOf(v.Value).Elem().SetBool(val)
+			case reflect.Slice:
+				// we assume that the slice here can be appended to (ex. an empty slice)
+				reflect.AppendSlice(reflect.ValueOf(v.Value), reflect.ValueOf(strings.Split(envValue, ",")))
```

Comment:
Maybe something like this would work? https://go.dev/play/p/DhbDNc4H-PL
```suggestion
				value := reflect.ValueOf(v.Value).Elem()
				value.Set(reflect.AppendSlice(value, reflect.ValueOf(strings.Split(envValue, ","))))
```